hack/codegen.sh

```diff
@@ -19,11 +19,11 @@ bandwidth() {
 }
 
 pricing() {
-  GENERATED_FILE="pkg/providers/pricing/zz_generated.pricing.go"
-  NO_UPDATE=$' pkg/providers/pricing/zz_generated.pricing.go | 4 ++--\n 1 file changed, 2 insertions(+), 2 deletions(-)'
+  GENERATED_FILE="pkg/providers/pricing/zz_generated.pricing_us.go"
+  NO_UPDATE=$' pkg/providers/pricing/zz_generated.pricing_us.go | 4 ++--\n 1 file changed, 2 insertions(+), 2 deletions(-)'
   SUBJECT="Pricing"
 
-  go run hack/code/prices_gen.go -- "${GENERATED_FILE}"
+  go run hack/code/prices_gen.go --region us --output "${GENERATED_FILE}"
```

Comment:
```suggestion
  go run hack/code/prices_gen.go --partition us --output "${GENERATED_FILE}"
```

I think we are actually talking about the partition here. These are the partition names, it's something to think about if we should just match the file naming and our CLI arg alignment with the well-known partition naming: https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html

Reply:
Seems reasonable, I can split it out into the three partitions (`aws`, `aws-cn`, and `aws-us-gov`) rather than `us` and `cn`.